src/Transform/ONNX/ShapeInferencePass.cpp

```diff
@@ -166,8 +154,12 @@ class ShapeInferencePass : public mlir::PassWrapper<ShapeInferencePass,
    */
   static bool returnsDynamicShape(Operation *op) {
```

Comment:
The naming of this function is vague. I think according to the current definition, it is better to name this function as "hasExplicitResultShape", and negate all usages of the original function output.